psvpfsparser/FilesDbParser.cpp

```diff
@@ -364,7 +366,7 @@ bool FilesDbParser::parseFilesDb(std::ifstream& inputStream, std::vector<sce_ng_
          std::vector<std::uint8_t> unusedData1(nUnusedSize1);
          inputStream.read((char*)unusedData1.data(), nUnusedSize1);
 
-         if(!isZeroVector(unusedData1))
+         if(!isZeroVector(unusedData1) && !is_bad_block)
```

Comment:
Please add another check for is_bad_block == true.
Print a warning so that we can keep track of the errors in console.
I understand that this code is implemented in such a way that it skips invalid data, but user needs at least to know about it.

Reply:
added separate check and warning

https://github.com/motoharu-gosuto/psvpfstools/blob/94471aefbc4e991bc2981f1489237df9c7ff44f2/psvpfsparser/FilesDbParser.cpp#L377-L381

Reply:
perfect